trace.go

```diff
@@ -206,27 +195,20 @@ func SetupTraceOTEL(ctx context.Context, optFns ...TraceOption) (tp *trace.Trace
 		)
 	}
 
-	tp = trace.NewTracerProvider(providerOpts...)
-	shutdownFuncs = append(shutdownFuncs, tp.Shutdown)
-
-	// This option usually needs to be set to false, if we need t test the code
-	// in production environment, we need to set this to true using WithDefaultTracer
-	if opt.setDefaultTracer {
-		otel.SetTracerProvider(tp)
-	}
```

Comment:
I think we need to bring this one back as I want the library does all the configuration for me so I don't have to remember to set it in my main program

Reply:
ok cool i am fine with it